### PR TITLE
added findElement test section

### PR DESCRIPTION
**Pull request by Ad1B3nAr0ya**

### CODE REVIEW

File: test/specs/FindElementSpec.ts [hunk 1]
```diff
@@ -3,15 +3,17 @@ import { BrowserUtils } from '../../src/commons/BrowserUtils';
 import { SelectorType } from '../../src/enums/SelectorType';
 import { describeCommon } from '../TestHelper';
 
+const IMAGE_XPATH: string = "//*[@id='content_images']/div/p/img";
 /**
  * wdio-allure-ts FindElement & FindElements tests
  */
 describeCommon('findElement of BrowserUtils Tests', () => {
```
Comment (FelixZilber) [inline test/specs/FindElementSpec.ts]: missing negative test -not existing element 
and test with multiple elements with same ID

Comment (Ad1B3nAr0ya) [reply test/specs/FindElementSpec.ts]: done

File: test/specs/FindElementSpec.ts [hunk 2]
```diff
@@ -16,4 +18,22 @@ describeCommon('findElement of BrowserUtils Tests', () => {
...
+  it('Fail to obtain element due to wrong path', () => {
+    const result: string = BrowserUtils.findElement(
+      SelectorType.XPATH,
+      BAD_XPATH
+    );
+
+    expect(JSON.stringify(result).toLowerCase()).not.contains('{"element');
+  });
+
+  it.only('First Element Returned', () => {
```
Comment (FelixZilber) [inline test/specs/FindElementSpec.ts]: remove only

Comment (Ad1B3nAr0ya) [reply test/specs/FindElementSpec.ts]: Done

**Pull request merged**